### ISSUE TITLE
Cannot reproduce teaser results

### ISSUE THREAD

**Issue by sincostanx** [RotsteinNoam/Paint-by-Inpaint#3]
Thank you for your great work!

I am trying to reproduce your teaser results but couldn't get it to work even after trying 10 different random seeds. 
Could you share the hyperparameters you used to generate these results?

**Expected results**
![image](https://github.com/user-attachments/assets/af80c461-0dc2-40d5-a63f-fa906942524d)

**Output**
![simple_reproduce1](https://github.com/user-attachments/assets/cdf2e5cd-998d-4139-9147-3c0753a15945)
![simple_reproduce2](https://github.com/user-attachments/assets/fda52ab2-6c32-4bd1-945c-163f3fd87da5)

**Code**
```python
from diffusers import StableDiffusionInstructPix2PixPipeline, EulerAncestralDiscreteScheduler
import torch
from PIL import Image
import os
from pathlib import Path
import numpy as np

def load_512(image_path: str, left=0, right=0, top=0, bottom=0):
    image = np.array(Image.open(image_path))[:, :, :3]
    h, w, c = image.shape
    left = min(left, w - 1)
    right = min(right, w - left - 1)
    top = min(top, h - left - 1)
    bottom = min(bottom, h - top - 1)
    image = image[top : h - bottom, left : w - right]
    h, w, c = image.shape
    if h < w:
        offset = (w - h) // 2
        image = image[:, offset : offset + h]
    elif w < h:
        offset = (h - w) // 2
        image = image[offset : offset + w]
    image = np.array(Image.fromarray(image).resize((512, 512)))
    return image 

pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(
    "paint-by-inpaint/add-base", torch_dtype=torch.float16, safety_checker=None
).to("cuda")
pipe.scheduler = EulerAncestralDiscreteScheduler.from_config(pipe.scheduler.config)

# image = Image.fromarray(load_512("./test_images/test5.jpg"))
# prompt = "Add a white buttoned shirt"

image = Image.fromarray(load_512("./test_images/test3.jpeg"))
prompt = "Add a big flat TV"

for i, seed in enumerate(range(10)):
    generator = torch.Generator(device="cuda").manual_seed(seed)

    out_images = pipe(
        prompt,
        image=image, 
        guidance_scale=7, 
        image_guidance_scale=1.5, 
        num_inference_steps=50,
        num_images_per_prompt=1,
        generator=generator,
    ).images[0]

    # save_path = os.path.join("./simple_reproduce", f"{i}.png")
    save_path = os.path.join("./simple_reproduce2", f"{i}.png")
    os.makedirs(Path(save_path).parent, exist_ok=True)
    out_images.save(save_path)
```

**Comment by RotsteinNoam** [comment]
Thank you for your kind words and interest in our work!

Our work includes several models, particularly for adding objects:
"paint-by-inpaint/add-base"
"paint-by-inpaint/add-finetuned-mb"
The results of these models tend to vary, and the choice of which to use should be based on your specific task. Additionally, the parameters guidance_scale and image_guidance_scale can be adjusted according to your needs.

For instance, in the case of the Spiderman image, a significant modification is needed. Therefore, by using a high guidance_scale value (10) and a low image_guidance_scale value (1.01) with the finetuned-mb model and your script, you will achieve the following results with the first seed:
![spider_shirt](https://github.com/user-attachments/assets/9d35d945-e025-4384-aded-23f970886adb)

**Comment by sincostanx** [comment]
Thanks, it works fine on my end!

Actually, I'd like to run your models on a large internal dataset, so manually adjusting the hyperparameters isn't practical.
Could you share the default hyperparameters you used in the quantitative experiments in your paper for each model (Sections 5.2 and 6)?

**Comment by navvewas** [comment]
Sorry for the late reply; we missed this message.

There are three different models in the sections you referred to. While all of them have the same Text guidance scale (7), the first model—Ours (without FT), corresponding to Tables 2, 3, and the upper part of Table 4—has an Image guidance scale of 1.5. The other two models—Ours FT on MagicBrush, corresponding to the lower part of Table 4, and Ours + IP2P FT, a model trained on both the PIPE dataset and IP2P data, fine-tuned on MagicBrush (Table 6)—have an Image guidance scale of 2.

Hope this helps! Let us know if you have any further questions.
